ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -413,6 +413,7 @@ public final class AliasTokens {
         case brandBackgroundInvertedDisabled
         case stencil1
         case stencil2
+        case canvasBackground
```

Comment:
are we making this alphabetical? what's the logic here? @laminemale

Reply:
The rest of the enum isn't alphabetical. There are comments grouping foreground and background colors so I added it at the end of the background colors group. Do we need to make the order in the enum alphabetical?

Reply:
I'm alright with it being alphabetical within the groupings, but it's genuinely arbitrary, so don't go rearranging things for no real need.

Reply:
it does make the searchability little better. how does design token figma file organize their order?